hc/settings.py

```diff
@@ -111,7 +115,7 @@
 USE_L10N = True
 USE_TZ = True
 
-SITE_ROOT = "http://localhost:8000"
+SITE_ROOT = "https://hc-gnext.herokuapp.com"
```

Comment:
Use link for group Heroku: https://dev-gnext.herokuapp.com/

Reply:
https://hc-gnext.herokuapp.com is the heroku link

Reply:
https://dev-gnext.herokuapp.com is the heroku link